plugins/node/opentelemetry-instrumentation-ioredis/README.md

```diff
@@ -59,11 +62,37 @@ It is also possible to define a custom serialization function. The function will
 Here is a simple example to serialize the command name skipping arguments:
 
 ```javascript
-const { IORedisInstrumentation } = require('@opentelemetry/instrumentation-ioredis');
+const {
+  IORedisInstrumentation,
+} = require("@opentelemetry/instrumentation-ioredis");
 
 const ioredisInstrumentation = new IORedisInstrumentation({
   dbStatementSerializer: function (cmdName, cmdArgs) {
     return cmdName;
+  },
+});
+```
+
+#### Using `requestHook`
+
+Instrumentation user can configure a custom "hook" function which will be called on every request with the relevant span and request information. User can then set custom attributes on the span or run any instrumentation-extension logic per request.
+
+Here is a simple example that adds a span attribute of `ioredis` instrumented version on each request:
+
+```javascript
+const { IORedisInstrumentation } = require('@opentelemetry/instrumentation-ioredis');
+
+const ioredisInstrumentation = new IORedisInstrumentation({
+requestHook: function (
+    span: Span,
+    requestInfo: IoRedisRequestHookInformation
+  ) {
+    if (requestInfo.moduleVersion) {
+      span.setAttribute(
+        "instrumented_library.version",
```

Comment:
```suggestion
        'instrumented_library.version',
```

Reply:
fixed. and I also fixed it in the 'db.statement` part of the README.
Thanks for spotting that